app/views/hub/metrics/index.html.erb

```diff
@@ -21,79 +21,172 @@
     </div>
   </div>
 
-  <div>
+  <div class="scrollable-table-wrapper">
     <table class="index-table org-metrics-table">
       <thead class="index-table__head">
         <tr class="index-table__row">
           <th scope="col" class="index-table__header sortable" id="organization-name" width="40%">
             Organization Name
           </th>
           <th scope="col" class="index-table__header sortable" id="needs-attention-breaches">
-            Needs attention breaches<sup>1</sup>
+            Attention breaches<sup>1</sup>
           </th>
+          <th scope="col" class="index-table__header" id="needs-attention-percentage">
```

Comment:
These columns aren't sortable currently, but since I set the data-js value in the js I can probably set it up to do so. I just dont want to add too much untested functionality all at once.